quic/s2n-quic-core/src/recovery/rtt_estimator.rs

```diff
@@ -280,6 +294,12 @@ impl RttEstimator {
         //# persistent congestion is established.
         self.first_rtt_sample = None;
     }
+
+    #[inline]
+    fn rttvar4x(&self) -> Duration {
```

Comment:
This method name looks a little weird to me, maybe `rttvar_4x`?

Reply:
that's fine

Reply:
https://github.com/aws/s2n-quic/pull/1493/commits/1bd8c1e719973ed94da50ee305fd7b5d4cf00392